manager/app/src/main/java/me/weishu/kernelsu/ui/screen/Settings.kt

```diff
@@ -151,6 +158,49 @@ fun SettingScreen(navigator: DestinationsNavigator) {
                 enableWebDebugging = it
             }
 
+            ListItem(
+                leadingContent = {
+                    Icon(
+                        Icons.Filled.Save,
+                        stringResource(id = R.string.save_log)
```

Comment:
We already have "send log". It is better to merge these two.

Reply:
ok got it

Reply:
But having this feature inside this app makes it more convenient and then user don't need to install any other apps just to do this simple thing